backend/api_calls.py

```diff
@@ -280,6 +279,10 @@ def AddPlayer(request, game_state):
     }
   })
 
+  # TODO: Maybe also check for duplicate names in the same game?
+  if ' '  in request['name']:
+    raise InvalidInputError('Name cannot contain spaces.')
```

Comment:
if you would be so kind, could you require that they not be named "all" here? (and in updateplayer)